spec/requests/api/requests_spec.rb

```diff
@@ -248,6 +248,31 @@
       expect(response.parsed_body).to match(expected)
       expect(response).to have_http_status(:ok)
     end
+
+    it "exposes allowed_tags in the request resources" do
```

Comment:
I'm a bit divided over this test, although it is written perfectly well! I'm just thinking that since we *should* have ample coverage that the API can deliver virtual attributes, is it needed? Perhaps a model test would suffice?